CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessViewCategoryPanelController.java

```diff
@@ -96,8 +101,22 @@ public boolean isChanged() {
         final Preferences prefs = NbPreferences.forModule(DataAccessViewPreferenceKeys.class);
         final DataAccessViewCategoryPanel panel = getPanel();
         final List<String> hiddenCategory = panel.getHiddenCategory();
-        return (!hiddenCategory.isEmpty())
-                || (!prefs.get(DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW, DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW_DEFAULT).isEmpty());
+        final List<String> visibleCategory = panel.getVisibleCategory();
+
+
+        if (reorderButtonPressed) {
+            if (!visibleCategory.equals(visibleNow)) {
+                orderChanged = true;
+                visibleNow = panel.getVisibleCategory();
+            } else {
+                orderChanged = false;
+                reorderButtonPressed = false;
+            }
+        }
+
+        return !hiddenCategory.isEmpty() || orderChanged
+                || !prefs.get(DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW, DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW_DEFAULT).isEmpty()
+                || !prefs.get(DataAccessViewPreferenceKeys.VISIBLE_DA_VIEW, DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW_DEFAULT).isEmpty();
```

Comment:
The logic here needs to be revised.
It should be checking the current visible list against the entries that were initially in the list when the options window was opened.
```
    return orderChanged || visibleEntriesHaveChanged();
```